lib/server.js

```diff
@@ -225,6 +242,13 @@ module.exports = function createServer(config, log) {
               retryAfter: limits.rateLimitIntervalSeconds,
               unblock: false
             })
+
+            if (config.reputationService.enable) {
```

Comment:
We might need to add test cases to cover the error blocks. Looks like some of these were not being covered and dropped coverage below 94%. https://coveralls.io/builds/8903869/source?filename=lib%2Fserver.js